cosmos-redis/config/redis_ephemeral.conf

```diff
@@ -0,0 +1,4 @@
+appendonly no
+save ""
+port 6380
```

Comment:
The existing redis.conf doesn't have a port specified ... I assume its the default?

Reply:
6379 is the default